src/jobs/models/jobManager.js

```diff
@@ -54,14 +57,16 @@ module.exports.createJob = async (job) => {
     try {
         await databaseConnector.insertJob(jobId, job);
         logger.info('Job saved successfully to database');
-        const latestDockerImage = await dockerHubConnector.getMostRecentRunnerTag();
-        const runId = Date.now();
-        const jobSpecificPlatformRequest = await createJobRequest(jobId, runId, job, latestDockerImage, configData);
+        const runId = uuid.v4();
```

Comment:
so this is ok that it's no longer Date.now()?

Reply:
runId === reportId === uuid.v4(), no DB migrations and no openapi validation on string.